elpis/wrappers/inference/gmm-decode-long.sh

```diff
@@ -34,16 +34,92 @@ echo "# dummy file" > ./conf/online_cmvn.conf
     exp/tri1 \
     exp/tri1_online
 
+# Assuming we are decoding a single utterance still
+
 # Manipulate the wav.scp file in the first (and only) split
-line=$(head -n 1 ./data/infer/spk2utt) && \
- utt=` echo ${line} | cut -d ' ' -f 2` &&
- echo "${utt} ./20030518Abui2PieterFrogDogBoy.wav" > ./data/infer/split1/1/wav.scp
+line=$(head -n 1 ./data/infer/spk2utt)
+utt=` echo ${line} | cut -d ' ' -f 2`
+spk=` echo ${line} | cut -d ' ' -f 1` # this was seg
+audio="audio.wav"
+length=`sox --i -D ${audio}`
+recid="decode"
+
+# Prepare the split dir
+splitDir=./data/infer/split1/
+if [ -d $splitDir ]; then rm -r $splitDir; fi
```

Comment:
As I've learnt from my extensive immersion in bash these holidays, the `[[ -d $splitDir ]]` format is safer to use with modern bash.

Reply:
ok, fixed in 654dbf2